src/background/services/mnemonic/index.ts

```diff
@@ -0,0 +1,8 @@
+import * as bip39 from "bip39";
+
+export const generateMnemonic = (): string => bip39.generateMnemonic();
+
+export const validateMnemonic = (mnemonic: string): boolean => bip39.validateMnemonic(mnemonic);
+
+export const mnemonicToSeed = (mnemonic: string): Promise<string> =>
+  bip39.mnemonicToSeed(mnemonic).then((bytes) => bytes.toString("hex"));
```

Comment:
```suggestion
import { generateMnemonic, validateMnemonic, mnemonicToSeed } from "bip39";

export const bip39GenerateMnemonic = (): string => generateMnemonic();

export const bip39ValidateMnemonic = (mnemonic: string): boolean => validateMnemonic(mnemonic);

export const bip39MnemonicToSeed = (mnemonic: string): Promise<string> =>
  mnemonicToSeed(mnemonic).then((bytes) => bytes.toString("hex"));
```

Reply:
I don't think that this solution works. If we have another version of mnemonic generation, I'd keep generic names without implementation details.

Reply:
Resolved because webpack will resolve this by tree-shaking.